src/Umbraco.Web/Umbraco.Web.csproj

```diff
@@ -531,39 +530,26 @@
     <Compile Include="Runtime\WebRuntimeComponent.cs" />
     <Compile Include="Editors\PublishedStatusController.cs" />
     <Compile Include="Editors\NuCacheStatusController.cs" />
-    <Compile Include="_Legacy\Actions\Action.cs" />
-    <Compile Include="_Legacy\Actions\ActionAssignDomain.cs" />
-    <Compile Include="_Legacy\Actions\ActionBrowse.cs" />
-    <Compile Include="_Legacy\Actions\ActionChangeDocType.cs" />
-    <Compile Include="_Legacy\Actions\ActionCollection.cs" />
-    <Compile Include="_Legacy\Actions\ActionCollectionBuilder.cs" />
-    <Compile Include="_Legacy\Actions\ActionCopy.cs" />
-    <Compile Include="_Legacy\Actions\ActionCreateBlueprintFromContent.cs" />
-    <Compile Include="_Legacy\Actions\ActionDelete.cs" />
-    <Compile Include="_Legacy\Actions\ActionEmptyTranscan.cs" />
```

Comment:
Is `ActionEmptyTrashCan` no longer needed?

Reply:
Makes sense why they are gone from Shan's previous comments.

An Action is no longer 1:1 tied to a menu item like it has previously been in the past. An Action is used to assign permissions & or for notifications